glusterd2/volgen/generate.go

```diff
@@ -49,7 +49,7 @@ func generateClusterLevelVolfiles(clusterinfo []*volume.Volinfo, xopts *map[stri
 			if err != nil {
 				return err
 			}
-			err = save(volfile.FileName, volfiledata)
+			err = save(volfile.FileName, volfiledata, "-")
```

Comment:
Passing `-` might be slightly confusing here.

You can have volfile type (as enums) and pass that may be ?

Suggestion:
Also, except for user mounting a volume usecase, we have tight control over volfile ID. This is because we pass `volfile-id` as an arg to all daemons. So we can come up with a common convention to name volfiles as we wish to make code simpler.

Reply:
This change is not necessary if I add `<volname>` prefix to all volfiles.